openapi/components/schemas/pastoral-item.yaml

```diff
@@ -0,0 +1,243 @@
+title: a Pastoral Record
+description: A pastoral record
+readOnly: true
+type: array
+properties:
+  id:
+    $ref: ./id.yaml
+  externalId:
+    description: The ID of SIS data corresponding to this pastoral record, if External Pastoral Synchronisation is ON
+    type: string
+    nullable: true
+  student:
+    $ref: ./userShort.yaml
+    _links:
+      type: object
+      properties:
+        profile:
+          type: string
+          format: uri-reference
+        avatar:
+          type: string
+          format: uri-reference
+          nullable: true
+  heading:
+    description: Heading of the pastoral record
+    type: string
+  type:
+    description: Type of the pastoral record
+    type: object
+    properties:
+      id:
+        description: The ID of the pastoral type associated to this pastoral record
+        type: integer
+      name:
+        description: The name of the pastoral type
+        type: string
+      colour:
```

Comment:
it doesn't look like we have colours for pastoral types or sub types... is this future planning?

Reply:
we do...

Reply:
I thought it was just for Severity ??

Reply:
Colour of icons.